core/build.gradle

```diff
@@ -39,6 +39,10 @@ dependencies {
 
     // Bouncy Castle
     compile 'org.bouncycastle:bcprov-jdk15on'
+
+
```

Comment:
nit: Could remove this extra line?

Reply:
@anuraaga ^^

Reply:
Thanks for the reminder - too many PRs ><